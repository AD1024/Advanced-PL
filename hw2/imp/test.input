n := 4;
x := 0;
y := n;
while 0 < y {
  x := x + 1;
  y := y - 1
};
assert x = n
