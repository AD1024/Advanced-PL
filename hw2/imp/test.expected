[("y", (Syntax.VInt 0)); ("x", (Syntax.VInt 4)); ("n", (Syntax.VInt 4))]
